Software-Engineering.md

```diff
@@ -1,163 +0,0 @@
-- I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
-- I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
-
-## Principal Software Engineer
-
-> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
-
-### I uncover important problems and opportunities that are critical to the long-term success of Octopus
-
-- I have a deep understanding of Octopus products and strategy
-- I collaborate with other Engineering leaders to ensure I am aware of all major initiatives at Octopus and can account for them in my own initiatives
-- I have a track record of identifying significant problems, inhibitors and opportunities, and for creating lasting, best-fit solutions that align our strategy and Engineering efforts
-- I work closely with product, engineering, and executive stakeholders to articulate and refine solutions to problems that are timely, important, and impactful. 
-- Instead of simply patching or extending existing solutions, which may not be fit for purpose anymore, my contributions open up whole new areas of strategic possibility that progress our journey of making Octopus a world-class Engineering organization.
-
-### I am the custodian of an area of strategic importance to Octopus
-
-- I am comfortable taking on large-scale greenfields and brownfields initiatives (e.g. re-architecting Octopus Server to allow for adding new step types as modular extensions, or creating a completely new Octopus Cloud platform)
-- I identify people and teams with the necessary skills, work out how to fill gaps, and guide and influence these people and teams collaboratively
-- I use a variety of tools and techniques to effectively communicate short and long-term plans and goals to my teams, stakeholders, Octopus senior leadership, and the broader organization
-- I use my experience to collaboratively steer teams toward good long-term technical decisions
-- The results of the initiatives I champion are measurable
-
-### I share my knowledge both within Octopus, and with the Octopus and wider developer community
-
-- I actively influence engineering teams across Octopus resulting in improved engineering excellence and overall productivity
-- I actively blog about the lessons we are learning as we improve engineering excellence to clarify our thinking, reinforce the learning, attract customers and potential future employees
-- I have a proven track record of expertly tailoring my communication to the target audience with a result of increased clarity and impact
-
-## Tech Lead / Engineering Manager
```

Comment:
New Engineering Manager role definitions have shipped in https://github.com/OctopusDeploy/People/pull/37, and as discussed in https://github.com/OctopusDeploy/People/pull/37#discussion_r728574148:

> On the old TL/EM role described here: We'll remove this old definition entirely and replace it with the updated, separate roles for technical and management tracks... Where there is a need for people to simultaneously take on technical and management responsibilities in future (for example, when seeding and growing a new team), Octonauts should generally still identify the primary track that the person is growing along in the longer term, and use that role definition.